builtin/receive-pack.c

```diff
@@ -249,6 +249,7 @@ static void show_ref(const char *path, const struct object_id *oid)
 			strbuf_addf(&cap, " push-cert=%s", push_cert_nonce);
 		if (advertise_push_options)
 			strbuf_addstr(&cap, " push-options");
+		strbuf_addf(&cap, " object-format=%s", the_hash_algo->name);
```

Comment:
@dscho It looks like some of your work on the hashing algorithm made it in. Is this a problem?

Reply:
These patches come from https://github.com/bk2204/git/compare/git:v2.27.0...transition-stage-4. I merged that branch at the end so that I can test the SHA-256 stuff: t5515 contains hard-coded commit hashes that depend on the main branch name, and I wanted to update the recorded hashes accordingly (and make sure that nothing else in the SHA-256 work breaks, either).

If you look at the commit history of the `inclusive-naming` branch, you will see that the tip commit is that merge commit. Obviously, I don't want to contribute those patches in the context of making Git's naming more inclusive. At the same time, I don't want to cause conflicts with that work, that's why I'll keep this merge commit on the top of this branch.